freeipa.spec.in

```diff
@@ -503,6 +503,7 @@ Requires: python-custodia
 Requires: python-dns >= 1.11.1
 Requires: python-netifaces >= 0.10.4
 Requires: pyusb
+Requires: python-jinja2
```

Comment:
I had that before, but removed it when I couldn't get pylint to actually complain about the missing import. But maybe that was just because "import-error" is disabled (twice?) in pylintrc. Anyway, added as a fixup commit.